docs/editing.md

```diff
@@ -41,6 +41,21 @@ At this point, the handling editor's job is to identify reviewers who have suffi
 
 To recruit reviewers, the handling editor can mention them in the `PRE-REVIEW` issue with their GitHub handle, ping them on Twitter, or email them. After expressing initial interest, candidate reviewers may need a longer explanation via email. See sample reviewer invitation email, below.
 
+**Potential ways to find reviewers**
+
+Finding reviewers can be challenging, especially if a submission is outside of your immediate area of expertise. Some strategies you can use to identify potential candidates:
+
+- Search the [reviewer spreadsheet](https://bit.ly/joss-reviewers) of volunteer reviewers. When using this spreadsheet, pay attention to the number of reviews this individual is already doing to avoid overloading them.
```

Comment:
I used to just CTRL-F through this spreadsheet like a crazy person until I realized I should be filtering the spreadsheet view with the Temporary Filter View option which allows one to filter two columns at once. Maybe it's obvious, but if not, here's a possible way to phrase this. (If it's obvious to everyone else, then there's no need to add this comment here.)

```suggestion
- Search the [reviewer spreadsheet](https://bit.ly/joss-reviewers) of volunteer reviewers. 
  - When using this spreadsheet, pay attention to the number of reviews this individual is already doing to avoid overloading them.
  - It can be helpful to use the "Data > Filter Views" capability to temporarily filter the table view to include only people with language or domain expertise matching the paper. 
```